docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/instrument-modeling/date-utilities.rst

```diff
@@ -283,3 +283,17 @@ the schedule end date and the payment frequency:
   :language: daml
   :start-after: -- CALCULATE_ACTACTISDA_DCF_SCHEDULE_PERIOD_BEGIN
   :end-before: -- CALCULATE_ACTACTISDA_DCF_SCHEDULE_PERIOD_END
+
+Summary
+=======
+
+You have learnt about the date-related utility functions in the Daml Finance library.
+Here are some pointers regarding next steps, depending on what you want to do:
+
+- understand and validate the implementation of existing Daml Finance instruments: the
+  implementation of the Bond package could be a good starting point. It is explained and linked
+  in the :doc:`Contingent Claims Instrument <contingent-claims-instrument>` tutorial.
```

Comment:
I think I gave a different title to that page when I reviewed the advanced topics section --> we should probably reflect that title